src/components/Button/InstantButton.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+
+import { Button } from '@material-ui/core'
+import styled from 'styled-components'
+
+export interface CustomButtonProps {
+    bgcolor?: string
```

Comment:
pls put it in camelCase

Reply:
> "Warning: React does not recognize the `bgColorHover` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `bgcolorhover` instead"
I think there is something to fix with the two styling props because "composite components should “consume” any prop that is intended for the composite component and not intended for the child component" https://reactjs.org/warnings/unknown-prop.html

Reply:
i forgot :+1: